airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -55,6 +55,24 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
     You can also pass additional keyword arguments like ``aws_secret_access_key``, ``aws_access_key_id``
     or ``region_name`` to this class and they would be passed on to Boto3 client.
 
+    There are two ways of storing secrets in Secret Manager for using them with this operator:
+    storing them as a conn URI in one field, or taking advantage of native approach of Secrets Manager
+    and storing them in multiple fields. There are certain words that will be searched in the name
+    of fileds for trying to retrieve a connection part. Those words are:
+
+    .. code-block:: ini
+
+        possible_words_for_conn_fields = {
+                'user': ['user', 'username', 'login', 'user_name'],
+                'password': ['password', 'pass', 'key'],
+                'host': ['host', 'remote_host', 'server'],
+                'port': ['port'],
+                'schema': ['database', 'schema'],
+                'conn_type': ['conn_type', 'conn_id', 'connection_type', 'engine'],
+            }
```

Comment:
Why is this ini?

Reply:
I just thought it was the standard way to define a code block in the doc. What should I put there instead?